documentation/src/docs/asciidoc/user-guide/running-tests.adoc

```diff
@@ -156,43 +148,46 @@ for a comprehensive list of options.
 [[running-tests-build-gradle-bom]]
 ===== Aligning dependency versions
 
+TIP: See <<running-tests-build-spring-boot>> for details on how to override the version
+of JUnit used in your Spring Boot application.
+
 Unless you're using Spring Boot which defines its own way of managing dependencies, it is
 recommended to use the JUnit Platform BOM to align the versions of all JUnit 5 artifacts.
 
 [source,groovy,indent=0]
 [subs=attributes+]
+.Explicit platform dependency on the BOM
 ----
 dependencies {
 	testImplementation(platform("org.junit:junit-bom:{bom-version}"))
+    testImplementation("org.junit.jupiter:junit-jupiter")
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
 }
```

Comment:
We should consistently use tabs instead of 4 spaces for indentation in all affected snippets in this PR.

Reply:
Good catch! Done in 1a45f17813d102746c1e9797b57cc03159d7e414